### PR TITLE
feat: beter Alt+Tab window switcher

### PR DESCRIPTION
**Pull request by ZhongYic00**
features include:
- indicator
  - win11-like equal-height-grid layout of window thumbnails
  - indicator follows screen not whole virtual desktop scene
  - tap & scroll interaction of the indicator
- current window preview
  - opaque not-focused windows
  - fix the bug that windows still receiving input when switcher shows

### CODE REVIEW

File: src/treeland/quick/qml/EQHGrid.qml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2023 Dingyuan Zhang <zhangdingyuan@uniontech.com>.
```
Comment (justforlxz) [inline src/treeland/quick/qml/EQHGrid.qml:1]: 名字是创建者的。

**Review by justforlxz — CHANGES_REQUESTED** [review]

**Comment by ZhongYic00** [comment]
recheck

File: src/treeland/quick/qml/EQHGrid.qml (hunk 1 above)
Comment (ZhongYic00) [reply src/treeland/quick/qml/EQHGrid.qml]: corrected with https://github.com/linuxdeepin/treeland/pull/166/commits/65d8c76bb8971f71b71bc47e4a543130b4fed079

**Review by wineee — CHANGES_REQUESTED** [review]

**Review by wineee — CHANGES_REQUESTED** [review]

**Review by wineee — APPROVED** [review]

File: src/treeland/quick/qml/WindowsSwitcher.qml [hunk 2]
```diff
@@ -79,37 +86,144 @@ Item {
...
+            const coord=parent.mapFromItem(activeOutput, 0, 0)
+            return coord.y
+        }
+        width: activeOutput?.width
+        height: activeOutput?.height
+        
+        MouseArea {
+            anchors.fill: parent
+            onClicked: {
+                root.visible = false
```
Comment (wineee) [inline src/treeland/quick/qml/WindowsSwitcher.qml:115]: ping

**Review by wineee — CHANGES_REQUESTED** [review]

File: src/treeland/quick/qml/WindowsSwitcher.qml (hunk 2 above)
Comment (ZhongYic00) [reply src/treeland/quick/qml/WindowsSwitcher.qml:115]: will fix keyboard interaction in another PR

**Comment by ZhongYic00** [comment]
recheck

**Review by justforlxz — APPROVED** [review]

**Pull request merged**